From 01a4256d74231925849dcfdb2b295fec3db386a2 Mon Sep 17 00:00:00 2001
From: Hendrik Brueckner <brueckner@redhat.com>
Date: Thu, 10 Jul 2014 09:13:08 -0400
Subject: [redhat] Kconfig: adapt watchdog device driver kernel config options

Message-id: <1404983588-14286-4-git-send-email-brueckner@redhat.com>
Patchwork-id: 85651
O-Subject: [RHEL7.1 PATCH 3/3] [redhat] s390/Kconfig: adapt watchdog device driver kernel config options
Bugzilla: 1088540
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Description
-----------
Adapt the kernel configuration option for the s390 watchdog
which has been adapted to use the generic watchdog API.

Bugzilla
--------
BZ 1088540
https://bugzilla.redhat.com/show_bug.cgi?id=1088540

Upstream status of the patch
----------------------------
This change applies to the RHEL7 kernel configuration only.

Brew
----
https://brewweb.devel.redhat.com/taskinfo?taskID=7683420

Test status
-----------
The patch has been tested and fixes the problem.
The fix has been verified by the IBM test department.

Signed-off-by: Hendrik Brueckner <brueckner@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/s390x/CONFIG_DIAG288_WATCHDOG b/redhat/configs/generic/s390x/CONFIG_DIAG288_WATCHDOG
new file mode 100644
index 0000000..b5287c5
--- /dev/null
+++ b/redhat/configs/generic/s390x/CONFIG_DIAG288_WATCHDOG
@@ -0,0 +1 @@
+CONFIG_DIAG288_WATCHDOG=m
diff --git a/redhat/configs/generic/s390x/CONFIG_ZVM_WATCHDOG b/redhat/configs/generic/s390x/CONFIG_ZVM_WATCHDOG
deleted file mode 100644
index abb8e63..0000000
--- a/redhat/configs/generic/s390x/CONFIG_ZVM_WATCHDOG
+++ /dev/null
@@ -1 +0,0 @@
-CONFIG_ZVM_WATCHDOG=m
-- 
1.7.1